### PR TITLE
Adds testimonial video to landing page

### PR DESCRIPTION
**Pull request by owocki**
Adds this testimonial video to the landing page:
https://www.youtube.com/watch?v=hpqHJCiqXzA&feature=youtu.be

Screenshot:

screen shot 2018-03-18 at 7 13 42 pm

### CODE REVIEW

**Comment by owocki** [comment]
i've also put the video(s) up at https://github.com/gitcoinco/creative/tree/master/video

submitting this PR to put the idea on the table.

i know that @PixelantDesign had some concerns about the graphical style of this testimonial video... so we may want to hold off on putting this on the landing page and only use it in a limited content way until we have better avatar assets and an agreed upon video style guide.

File: app/retail/templates/shared/testimonials.html [hunk 1]
```diff
@@ -55,6 +55,12 @@ <h4 class="pt-1">Who's using Gitcoin?</h4>
     </div>
   </div>
 
+    <div class="row mt-2 pb-3 animatedParent" >
+        <div class="col d-flex flex-column align-items-center animatedParent youtube">
+            <iframe style="height:288px; width:500px; " src="https://www.youtube.com/embed/hpqHJCiqXzA" class="animated fadeInRightShort" frameborder="0" allowfullscreen></iframe>
+        </div>
+
```
Comment (thelostone-mc) [inline app/retail/templates/shared/testimonials.html]: @mbeacom Does our linter catch empty lines?

Comment (thelostone-mc) [reply app/retail/templates/shared/testimonials.html]: @mbeacom   This one :  "no-multiple-empty-lines": "error" 
Doc: [link](https://eslint.org/docs/rules/no-multiple-empty-lines)

If needed , I can shoot out a PR with this

Comment (mbeacom) [reply app/retail/templates/shared/testimonials.html]: @thelostone-mc Thanks - https://github.com/gitcoinco/web/commit/ef1d21418e4f517b9b40763437be21e13d5205a7

**Pull request merged**